src/pages/src/language/lang/en.js

```diff
@@ -457,4 +457,5 @@ export default {
   确认退出登录_: '确认退出登录？',
   _需要设置新密码: ', You need to set a new password',
   导入覆盖提升: 'After checking, it is allowed to edit the personal information of existing users, and the existing user information will face the risk of being modified, so be cautious.',
+  允许对同名用户覆盖更新: 'Allow users with the same name to overwrite updates',
```

Comment:
Allow override updates for users with the same name

Reply:
done